spec/controllers/news_controller_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'rails_helper'
+
+RSpec.describe NewsController, type: :controller do
+  describe 'GET #index' do
+    before { get :index }
```

Comment:
I think, it's unnecessary use before in this way..... @okohuch , @vikabelous  what do you think?